### PR TITLE
DON'T MERGE test config options

### PR DESCRIPTION
**Pull request by kimsauce**
@titusfortner per our convo, here's the Test Config Options first draft for your review. Need help determining which content is ancient vs. current!

**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/test-configuration-options.md [hunk 1]
```diff
@@ -0,0 +1,884 @@
...
+__Example__: `"browserVersion": "latest"`
+
+:::tip Default to Latest Version of Chrome, Firefox, or Microsoft Edge
+
+If you want to use the latest stable version of Google Chrome or Firefox that Sauce supports, you can use `"browserVersion": "latest"`. You can also use `"browserVersion": "latest-1"` or `"browserVersion": "latest-2"`, etc. to request the next most recent versions of a browser. For example, if the latest stable version of Chrome is 73, you can request `"latest-2"` to use Chrome 71.
+:::
+
+:::tip Microsoft Edge versions
+
+Microsoft Edge has two version numbers, the browser application version and the EdgeHTML rendering engine version. For example, the current stable release of Edge as of November 2019 has the browser application version 44.17763 and the EdgeHTML version 18.17763. For more details, see the [Microsoft Edge Wikipedia page](https://en.wikipedia.org/wiki/Microsoft_Edge).
```
Comment (kimsauce) [inline docs/test-configuration-options.md:45]: @titusfortner Comments from Diego and Michelle:

- In my opinion, we should replace this content with the new Edge versions, and also add a comment for IE and Safari.
- I don't think we have the need to keep this Edge information regarding legacy versions. Should just add edge to the above section about using "latest". I would like to possibly see something here or in the platformName section about using latest for Safari versions and the ability to not pin to a specific MacOS version in order to get the latest safari version even if it's on a new macOS version.

File: docs/test-configuration-options.md [hunk 2]
```diff
@@ -0,0 +1,884 @@
...
+
+</details>
+<br/>
+
+__Value Type__: string.
+
+__Example__: `"iedriverVersion": "3.141.0"`
+<br/>
+
+### `seleniumVersion`
```
Comment (kimsauce) [inline docs/test-configuration-options.md:183]: @titusfortner from Matt Dunn:

I think this whole section (`seleniumVersion`) needs a revamp. For example, the selenium version is ignored if using Chrome or Edge, as we don't use a selenium server at all. That's mentioned in a footnote here, but it's not very clear when looking at the working and table on the right.

File: docs/test-configuration-options.md (hunk 1 above)
Comment (titusfortner) [reply docs/test-configuration-options.md:45]: This is my proposed wording: 
https://github.com/saucelabs/sauce-docs/pull/368/files#diff-0fc2069221eb88630ed37f491c9da96f8a956c53a37c3a6f2cbb4c7e8d16cf70R44-R53

File: docs/test-configuration-options.md (hunk 2 above)
Comment (titusfortner) [reply docs/test-configuration-options.md:183]: Yeah, I got stalled out yesterday revamping this section (it's a beast). Maybe I'll save it for last to maintain momentum. :)

**Pull request merged**